LibSWBF2/Wrappers/Light.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "Chunks/LVL/lght/lght.h"
+#include "InternalHelpers.h"
+
+#include "Types/Colorf.h"
+#include "Types/Vector4.h"
+#include "Types/Vector3.h"
+
+
+namespace LibSWBF2::Wrappers
+{
+
+using namespace LibSWBF2::Chunks::LVL::lght;
+using namespace LibSWBF2::Types;
+
+struct Light {
+
+public:
+
+	String GetName();
+
+	Vector4 GetRotation();
+	Vector3 GetPosition();
+	ELightType GetType();
+	Vector3 GetColor();
+
+	bool GetCastSpecular();
+	bool GetRange(float_t& rangeOut);
+	bool GetSpotAngles(float_t& innerAngleOut, float_t& outerAngleOut);
```

Comment:
Getters for type-dependent fields (returns false if the Light doesn't have the requested field)